vllm/model_executor/models/mamba.py

```diff
@@ -346,18 +338,8 @@ def forward(
 
 
 class MambaForCausalLM(nn.Module, HasInnerState, IsAttentionFree):
-    packed_modules_mapping = {
-        "qkv_proj": [
-            "q_proj",
-            "k_proj",
-            "v_proj",
-        ],
-    }
-
     # LoRA specific attributes
     supported_lora_modules = [
-        "qkv_proj",
-        "o_proj",
         "embed_tokens",
         "lm_head",
```

Comment:
I don't see these modules in the checkpoint either https://huggingface.co/state-spaces/mamba-130m-hf?show_file_info=model.safetensors